tests/external_botocore/test_bedrock_chat_completion.py

```diff
@@ -92,11 +91,29 @@ def expected_events(model_id):
     return chat_completion_expected_events[model_id]
 
 
+@pytest.fixture(scope="module")
+def expected_events_no_content(model_id):
+    events = copy.deepcopy(chat_completion_expected_events[model_id])
+    for event in events:
+        if "content" in event[1]:
+            del event[1]["content"]
+    return events
+
+
 @pytest.fixture(scope="module")
 def expected_invalid_access_key_error_events(model_id):
     return chat_completion_invalid_access_key_error_events[model_id]
 
 
+@pytest.fixture(scope="module")
+def expected_invalid_access_key_error_events_no_content(model_id):
+    events = copy.deepcopy(chat_completion_invalid_access_key_error_events[model_id])
```

Comment:
Done!  For some reason I thought this was a shallow copy function.

Reply:
JK, looking more into this, it DOES create a shallow copy, but it creates a new reference for the dictionaries (instead of using the same one).  Either way, I need to use deepcopy for this so that the contents of the dict isn't changed